elfloader-tool/src/arch-arm/linker.lds

```diff
@@ -24,7 +18,6 @@ SECTIONS
         *(.text.start)
     }
 }
-INSERT BEFORE .text;
 
 SECTIONS
 {
```

Comment:
What is the new section table layout after these changes?

This is what currently seems to be produced:
```
  0 .start        00000034  85000000  85000000  00010000  2**2
                  CONTENTS, ALLOC, LOAD, READONLY, CODE
  1 .text         000073c4  85001000  85001000  00011000  2**12
                  CONTENTS, ALLOC, LOAD, READONLY, CODE
  2 .rodata       00000ec7  850083c4  850083c4  000183c4  2**2
                  CONTENTS, ALLOC, LOAD, READONLY, DATA
  3 .ARM.exidx    00000008  8500928c  8500928c  0001928c  2**2
                  CONTENTS, ALLOC, LOAD, READONLY, DATA
  4 .bss          00010048  8500c000  8500c000  00019294  2**14
                  ALLOC
  5 ._archive_cpio 01842a00  8501c048  8501c048  0002c048  2**0
                  CONTENTS, ALLOC, LOAD, DATA
  6 .data         0000002c  8686ea48  8686ea48  0187ea48  2**2
                  CONTENTS, ALLOC, LOAD, DATA
  7 _driver_list  00000030  8686ea74  8686ea74  0187ea74  2**2
                  CONTENTS, ALLOC, LOAD, DATA

```

Reply:
lld (unlike ld.bfd) does place .rodata before .text, the layout after the changes are:

`[ 1] .rodata.str1.1    PROGBITS        0000000040a11000 001000 000b45 01 AMS  0   0  1
  [ 2] .rodata           PROGBITS        0000000040a11b48 001b48 000668 00   A  0   0  8
  [ 3] .start            PROGBITS        0000000040a121b0 0021b0 00002c 00  AX  0   0  1
  [ 4] .text             PROGBITS        0000000040a13000 003000 007214 00  AX  0   0 4096
  [ 5] .bss              NOBITS          0000000040a1b000 00b000 006090 00  WA  0   0 4096
  [ 6] ._archive_cpio    PROGBITS        0000000040a21090 011090 486c00 00  WA  0   0  1
  [ 7] _driver_list      PROGBITS        0000000040ea7c90 497c90 000060 00 WAR  0   0  8
  [ 8] .data             PROGBITS        0000000040ea7cf0 497cf0 000150 00  WA  0   0  8`